src/System.Runtime.Extensions/tests/System/AppDomainTests.cs

```diff
@@ -310,6 +305,7 @@ public void Unload()
         }
 
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap | TargetFrameworkMonikers.UapAot, "Assembly.Load(byte[], ...) is not supported in AppX.")]
```

Comment:
@danmosemsft @AtsushiKan please use this thread: https://github.com/dotnet/coreclr/issues/9912